stack/stack.py

```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+
```

Comment:
Okay will make the changes back but it will continue to occur in other prs also then after this PR is merged as PEP8Bear will point it out in them. Will create a separate issue for that is it okay @singhpratyush sir?